python/lsst/ts/standardscripts/maintel/home_both_axes.py

```diff
@@ -83,21 +87,25 @@ async def configure(self, config):
         if self.mtcs is None:
             self.mtcs = MTCS(domain=self.domain, log=self.log)
             await self.mtcs.start_task
-        self.ignore_m1m3 = config.ignore_m1m3
+
+        if hasattr(config, "ignore_m1m3"):
+            warnings.warn(
```

Comment:
this warning will be printed to stdout (or stderr, I am not sure) but won't be logged. Please, add a `self.log.warning` call here as well.